_posts/2024/2024-01-15-the-first-five-questions.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "The First Five Questions"
+date: 2024-01-15
+year: 2024
+---
+
+I've been talking and writing for several years
+about the need for something that would teach programmers
+the useful basics of economics, politics, sociology, and other social sciences.
+A few days ago I framed it as [the Scratch of social sciences][like-scratch];
+some people interpreted that as, "What software would you use to teach these ideas?"
+but as I later wrote on Mastodon,
+when I say, "X is to the social sciences as Scratch is to computer science,"
+what I'm really looking for is something that combines approachability with a solid foundation
+in the way that Scratch does.
+
+One possibility is something like the [Carpentries][carpentries] workshops.
+A typical workshop spends half a day on the Unix shell,
+half a day on version control,
+and a full day on either Python or R;
+all of the lessons are aimed at people who have little or no previous training in programming
+(though they may often have picked up bits and pieces in a data analysis class).
+
+Could that format be used to teach a few actionable fundamentals of the social sciences?
```

Comment:
I think this is an excellent approach/idea and one that is consistent with the approach I noted by profs as a practical "what do you do if its a large group of people and not a special seminar series where everyone has time to get your feedback and help," where you define common experiences and the how and why questions associated with them, and walk through the process together.